CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+[0.39.5] - 2017-08-02
```

Comment:
as discussed in retro, this shouldn't be updated until the PR is ready to merge.